examples/shadowmap/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "luma.gl-examples-shadowmap",
+  "version": "0.0.0",
+  "description": "An es6 shadowmap example for luma.gl.",
+  "license": "MIT",
+  "author": "Ib Green <ib@uber.com>",
+  "dependencies": {
+    "babel-polyfill": "6.5.0",
+    "gl-format-compiler-error": "^1.0.2",
+    "global": "^4.3.0",
+    "glslify": "^5.0.2"
+  },
+  "browserify": {
+    "transform": [
+      "babelify",
+      "glslify"
+    ]
+  },
+  "devDependencies": {
+    "babel-cli": "6.5.1",
+    "babel-core": "6.5.1",
+    "babel-eslint": "4.1.8",
+    "babel-plugin-transform-builtin-extend": "1.1.0",
+    "babel-preset-es2015": "6.5.0",
+    "babel-preset-stage-2": "^6.5.0",
+    "babelify": "^7.2.0",
+    "browserify": "^13.0.0",
+    "budo": "^8.0.3"
+  },
+  "scripts": {
+    "build": "browserify src/index.js > bundle.js",
```

Comment:
What's the thinking here? Should the built version of this go into a local dist directory and be checked in so we can load this example directly from our gh-pages?